src/services/rates/acbuRateCache.ts

```
@@ -0,0 +1,40 @@
+import { prisma } from "../../config/database";
+import { AcbuRate } from "@prisma/client";
+
+const TTL_MS = 10_000; // 10 seconds
+
+let cached: AcbuRate | null = null;
+let expiresAt = 0;
+let inflight: Promise<AcbuRate> | null = null;
+
+async function fetchFromDb(): Promise<AcbuRate> {
+  const row = await prisma.acbuRate.findFirst({ orderBy: { timestamp: "desc" } });
```

ESLint

[error] Replace `·orderBy:·{·timestamp:·"desc"·}` with `⏎····orderBy:·{·timestamp:·"desc"·},⏎·`

(prettier/prettier)